bazel/external/clutter.BUILD

```diff
@@ -0,0 +1,105 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
Shouldn't this all be done via Gazelle since this is Go Bazel?

Reply:
maybe - ill check further to get better understanding

Reply:
@htuch @itayd im afraid this is beyond my `go` knowledge - im looking at gazelle docs and not seeing immediately how it can be used

my vague understanding is that it might be used to generate this file - but i dont see any comparable examples to work from

@htuch can you be more specific about what should change ?

Reply:
@itayd can you have Gazelle provide a Bazel BUILD system in Clutter?

Reply:
not really sure what this means - i'll check soon.